### PR TITLE
Add explanation of Accuweather API rate limit

### PR DESCRIPTION
**Pull request by tels7ar**
The accuweather api is extremely rate-limited for the free level.  Add an explanation so people aren't caught by surprise.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @tels7ar, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16198).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/accuweather.markdown [hunk 1]
```diff
@@ -27,3 +27,9 @@ To add AccuWeather to your installation, go to **Configuration** >> **Integratio
 Due to limitations of the terms of use of AccuWeather free API key, it is possible to configure only one integration instance.
 
 </div>
+
+<div class="note warning">
+
+The Limited Trial account only allows 50 API calls per day.
+
+</div>
```
Comment (frenck) [inline source/_integrations/accuweather.markdown]: Please combine it with the warning that is already there. We don't have to add a new warning box for this.

**Review by frenck — CHANGES_REQUESTED** [review]
Small comment, other than that 👍

**Comment by homeassistant** [comment]
Hi @tels7ar, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16198).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by frenck — APPROVED** [review]
Tweaked it myself. Thanks for the addition @tels7ar 👍

**Pull request merged**